draft-ietf-lake-authz.md

```diff
@@ -579,10 +579,14 @@ W verifies and decrypts ENC_U_INFO using the relevant algorithms of the selected
 
 In case OPAQUE_INFO is present, it is made available to the application.
 
-W calculates the hash of message_1 H(message_1), and associates this session identifier to the device identifier ID_U. If H(message_1) is not unique among session identifiers associated with this device identifier of U, the EDHOC session SHALL be aborted.
+W calculates the hash of message_1 H(message_1), and associates this session identifier to the device identifier ID_U. To avoid processing duplicates, H(message_1) SHALL be unique among session identifiers associated with a given ID_U.
+
```

Comment:
As in #28

Reply:
Answered [here in 28](https://github.com/lake-wg/authz/pull/28/files#r1613175478)